site/content/internal/infrastructure/kubernetes/kubernetes-troubleshooting.md

```diff
@@ -0,0 +1,102 @@
+---
+title: "Kubernetes Troubleshooting"
+date: 2018-11-07T15:24:42+01:00
+weight: 10
+---
+
+This page have the intent to help the developers access and perform any type of maintenance in the Production Mattermost Kubernetes Cluster which is running on AWS using the EKS.
+
+
+## Setup local environment to access K8s
+
+First if you don't have `kubectl` installed you will need to install, also if your `kubectl` is older then version 1.10 you need to update.
+
+Also you will need the AWS Keys for the Main Mattermost AWS account. You can get one using Onelogin, please follow this [instructions](../../onelogin-aws)
+
+### To install kubectl on MacOS
```

Comment:
Can we just link to the official install instructions for all platforms? https://kubernetes.io/docs/tasks/tools/install-kubectl/

Reply:
ok.
and please test it :)